addons/website_event/models/event.py

```diff
@@ -36,7 +38,7 @@ def _default_hashtag(self):
     show_menu = fields.Boolean('Dedicated Menu', compute='_get_show_menu', inverse='_set_show_menu',
                                help="Creates menus Introduction, Location and Register on the page "
                                     " of the event on the website.", store=True)
```

Comment:
As you are on it then why not change orders of fields and methods(According to guidelines)